src/sql/workbench/contrib/executionPlan/browser/executionPlanComparisonPropertiesView.ts

```diff
@@ -45,10 +55,11 @@ export class ExecutionPlanComparisonPropertiesView extends ExecutionPlanProperti
 		} else {
 			target = localize('executionPlanPropertiesEdgeOperationName', "Edge");
 		}
-		const titleText = localize('executionPlanComparisonPropertiesTopOperation', "Top operation: {0}", target);
-		this._topOperationNameContainer.innerText = titleText;
-		this._topOperationNameContainer.title = titleText;
-		this.addDataToTable();
+		this._leftTitleText = localize('executionPlanComparisonPropertiesLeftOperation', "Left operation: {0}", target);
```

Comment:
These seem like they should be static (or preferably constant) values. Is there a reason you're making them class members instead?

Reply:
The target value can vary and comes from the `InternalExecutionPlanElement` passed in, which is why I couldn't declare them as const because there's no way to know what the target value will be ahead of time.

Reply:
Yeah that sounds reasonable